src/tensor_ops.jl

```diff
@@ -152,17 +157,22 @@ Computes the trace of a second or fourth order tensor.
     @code :(return s)
 end
 
+const vol = trace
+
+
 ########
 # Mean #
 ########
+
 Base.mean{dim}(S::SecondOrderTensor{dim}) = trace(S) / dim
```

Comment:
I'm pretty sure that Julias own inliner will do that automatically but it doesn't hurt to add it manually :)